plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/metastore/TestDeltaLakeMetastoreStatistics.java

```diff
@@ -164,6 +164,7 @@ private DeltaLakeTableHandle registerTable(String tableName, String directoryNam
         return new DeltaLakeTableHandle(
                 schemaTableName.getSchemaName(),
                 schemaTableName.getTableName(),
+                true,
```

Comment:
```suggestion
                false,
```

We have an external table here.